src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partner = '';
```

Comment:
maybe better way to use ternary operator? 
Also use const instead let (this variable never changes)

Reply:
agree, use ternary here